platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManager.java

```diff
@@ -45,6 +47,8 @@
 
 import com.google.common.base.Stopwatch;
 
+import sun.security.krb5.internal.crypto.Des;
```

Comment:
Woah! Where the heck did that ?! ... ah; probably while I was trying to type **Des**cribable ;)